### DISCUSSION TITLE
Création graphique de tendance

### DISCUSSION THREAD

**Discussion by Synfactory** [witsa/synapps#776] [Q&A]
Bonjour,
j'aurai besoin d'un coup de main pour créer un graphique du type, je vois pas du tout comment m'y prendre : 
![image](https://github.com/user-attachments/assets/b27d2eb0-a50b-4677-86e3-9cbfeda374f4)
Deux échelles modifiables + 2 points de données (que j'ai en ressources).

Si vous avez des pistes :) Merci

**Comment by manufitoussiwit** [comment]
Salut, peut être que ceci pourrait vous mettre sur la voie : 

https://github.com/witsa/synapps/discussions/633

**Comment by Synfactory** [comment]
Merci. Mon souci est que les données sont envoyées toutes les minutes. (Jai un point de donnée toute les minutes pour x et un autre pour y).
Je cherche la meme chose que les graphiques intégrés dans le REDY mais en pouvant changer l'abscisse. (Que ca soit pas le temps).
